### PR TITLE
Prettify LaTeX PDF output

### PR DESCRIPTION
**Pull request by McSinyx**
Resolve GH-125 and [beautify other places](https://patchouli.sr.ht/builds.sr.ht/artifacts/~huyngo/401504/768bfb2fa9aecb1c/acanban.pdf).  Other than cosmetic changes, this defines the substitution `|~|` for non-breaking space (see diff for example usage).

### CODE REVIEW

File: docs/source/meth/choices.rst [hunk 1]
```diff
@@ -52,8 +53,10 @@ but built as methods in the implementing language.
...
-   retrieved 2021-01-18, URL: https://jinja.palletsprojects.com/en/2.11.x/templates/#html-escaping
-.. [rethinkdbdoc] RethinkDB, *Introduction to ReQL -- RethinkDB*,
-   retrieved 2021-01-18, URL: https://rethinkdb.com/docs/introduction-to-reql/#reql-embeds-into-your-programming-language
+.. _Jinja2: https://jinja.palletsprojects.com
+.. [jinja2esc] The Pallets Project.  `"Template Designer Documentation"
+   <https://jinja.palletsprojects.com/en/2.11.x/templates/#html-escaping>`_.
+   *Jinja Documentation (2.11.x)*.  Retrieved 2021-01-18.
+.. [rethinkdbdoc] RethinkDB. `"Introduction to ReQL"
+   <https://rethinkdb.com/docs/introduction-to-reql/#reql-embeds-into-your-programming-language>`_.
+   Retrieved 2021-01-18.
```
Comment (Huy-Ngo) [inline docs/source/meth/choices.rst:62]: Iirc we have to submit a printed copy for the report, on which these links won't be accessible.

Comment (McSinyx) [reply docs/source/meth/choices.rst:62]: Huh, that sucks.  Lemme try another way.

Comment (McSinyx) [reply docs/source/meth/choices.rst]: Addressed in 39d89d3.

**Review by Huy-Ngo — APPROVED** [review]

**Comment by Huy-Ngo** [comment]
image
There is a large space here but I don't think it's a big deal.

**Pull request merged**